### ISSUE TITLE
Loopback fast path is deprected on Windows and can cause high cpu on Win Server 2019

### ISSUE THREAD

**Issue by maksimkim** [mgravell/Pipelines.Sockets.Unofficial#58]
See deprecation statement: https://docs.microsoft.com/en-us/windows-hardware/drivers/network/sio-loopback-fast-path
After upgrade from Server 2016 to Server 2019 we noticed issue with high cpu usage by system.exe process. Windows Server support engineers claim that could be caused by some bug in loopback fast path implementation in tcp.sys. 
Given the feature is deprecated does it make sense to remove it from the source code?

https://github.com/mgravell/Pipelines.Sockets.Unofficial/blob/master/src/Pipelines.Sockets.Unofficial/SocketConnection.Connect.cs#L67

**Comment by mgravell** [comment]
Yes, with that document: I'm happy to remove that. It was actually added by
Microsoft, but: agree it should go now.

**Comment by mgravell** [comment]
planning to deploy this today - just fixing .NET 5 build here: https://github.com/mgravell/Pipelines.Sockets.Unofficial/pull/60
